projects/fmcomms5/zcu102/system_bd.tcl

```diff
@@ -0,0 +1,13 @@
+
+source $ad_hdl_dir/projects/common/zcu102/zcu102_system_bd.tcl
+
+#set_property -dict [list CONFIG.PCW_EN_CLK2_PORT {1}] $sys_ps7
```

Comment:
Two remarks here:
- Please delete all the commented code
- What is the reasoning to define another clock source in the system_bd.tcl? Couldn't you just use the sys_200m_clk or sys_cpu_clk for the DMA clock?

Thanks,
Istvan